examples/aiarena/example_bot.py

```diff
@@ -1,23 +0,0 @@
-import sc2
-
-class ExampleBot(sc2.BotAI):
-    def __init__(self):
-        # Improves bot performance by a little bit
-        self.raw_affects_selection = True
-        # The distance calculation method: 0 for raw python, 1 for scipy pdist, 2 for scipy cdist
-        self.distance_calculation_method = 2
-
-    async def on_step(self, iteration):
-        # Populate this function with whatever your bot should do!
-        pass
-
-    async def on_start(self):
-        print("Game started")
-        # On game start or in any frame actually, you can set the game step here - do not put it in the __init__ function as the client will not have been initialized yet
-        self.client.game_step = 2
```

Comment:
This should also be visible that in the python-sc2 API the default game_step is at 8, but can be lowered to make the bot's micro more responsive.

Reply:
Same deal as: https://github.com/BurnySc2/python-sc2/pull/41#discussion_r368477829
I'll work something out and submit for review.